website/app/main.js

```diff
@@ -18,7 +18,7 @@ window.MacMessenger = {
   showSettings: function() {
     this.openGearMenu();
     document.querySelector(
-      'div.uiLayer.uiContextualLayerPositioner#js_1 > div.uiContextualLayer a'
+      'div.uiLayer.uiContextualLayerPositioner#js_2 > div.uiContextualLayer a'
```

Comment:
Do you think we can fix the duplicate selectors and improve this regression? I was thinking about making the selector for the cog more unique (not location based though) - something like `[title*="Settings"]`. Then do a lookup based on that elements "aria-owns" like `document.getElementById(cog.getAttribute('aria-owns'))`.